communities/spoc/lab/sections/1_beginner.yml

```diff
@@ -60,128 +60,128 @@ tabs:
-#        inputs:
-#          - label: Single cell data
-#            datatypes:
-#              - hdf5 (AnnData)
-#        button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc/scanpy_plot%2fscanpy_plot"
-#        button_icon: run
-#        button_tip: Run tool
-#      - title_md: <code>Scanpy PlotEmbed</code>
-#        description_md: >
-#          This tool allows you to plot embeddings like UMAPs.
-#        inputs:
-#          - label: Single cell data
-#            datatypes:
-#              - hdf5 (AnnData)
-#        button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2fscanpy_plot_embed%2fscanpy_plot_embed"
-#        button_icon: run
-#        button_tip: Run tool
-      # - title_md: <code>Scanpy PlotTrajectory</code>
-      #   description_md: >
-      #     This tool allows you to plot trajectory data, such as PAGA, pre-calculated in an AnnData object.
-      #   inputs:
-      #     - label: Single cell data
-      #       datatypes:
-      #         - hdf5 (AnnData)
-      #   button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxi%2fscanpy_plot_trajectory/scanpy_plot_trajectory"
-      #   button_icon: run
-      #   button_tip: Run tool
-      # - title_md: <code>Scanpy Plot dimension reduction</code>
-      #   description_md: >
-      #     This tool allows you to plot embeddings such as PCA, UMAP, and tSNE.
-      #   inputs:
-      #     - label: Single cell data
-      #       datatypes:
-      #         - rds (Seurat object)
-      #         - rdata.sce (Single Cell Experiment)
-      #         - h5 (Seurat)
-      #         - hdf5 (Loom/AnnData)
-      #   button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2fseurat_dim_plot%2fseurat_dim_plot"
-      #   button_icon: run
-      #   button_tip: Run tool
-      # - title_md: <code>Plot with Seurat</code>
-      #   description_md: >
-      #     This tool allows you to plot gene expressions, such as with Violin Plots.
-      #   inputs:
-      #     - label: Single cell data
-      #       datatypes:
-      #         - rds (Seurat object)
-      #         - rdata.sce (Single Cell Experiment)
-      #         - h5 (Seurat)
-      #         - hdf5 (Loom/AnnData)
-      #   button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Febi-gxa%2fseurat_plot%2fseurat_plot"
-      #   button_icon: run
-      #   button_tip: Run tool
+  - id: data_viz
+    title: Visualising Data
+    heading_md: >
+      There are a few key tools for visualising single-cell data in Galaxy.
+    content:
+      - title_md: <code>Plot with Scanpy</code>
+        description_md: >
+          This tool allows you to generate many plots, from scatterplots, to violin plots, to gene heatplots.
+        inputs:
+          - label: Single cell data
+            datatypes:
+              - hdf5 (AnnData)
```

Comment:
Just a suggestion - rather than labelling all these datatypes repeatedly, wouldn't it make more sense to change the "Converting datatypes" tab to "Single cell datatypes" where the first item defines these properly? The `datatype` is supposed to be a Galaxy datatype, and adding labels in brackets makes that less clear.

![image](https://github.com/user-attachments/assets/aa7c194a-4d76-49c9-a37c-85ec3740d969)